modules/server/src/main/scala/scaladex/server/route/api/ArtifactApi.scala

```diff
@@ -0,0 +1,52 @@
+package scaladex.server.route.api
+
+import scala.concurrent.ExecutionContext
+
+import akka.http.scaladsl.server.Route
+import ch.megard.akka.http.cors.scaladsl.CorsDirectives.cors
+import endpoints4s.akkahttp.server
+import scaladex.core.api.artifact.ArtifactEndpoints
+import scaladex.core.api.artifact.ArtifactParams
+import scaladex.core.api.artifact.ArtifactResponse
+import scaladex.core.model.Language
+import scaladex.core.model.Platform
+import scaladex.core.model.search.Page
+import scaladex.core.model.search.Pagination
+import scaladex.core.service.WebDatabase
+
+class ArtifactApi(database: WebDatabase)(
+    implicit ec: ExecutionContext
+) extends ArtifactEndpoints
+    with server.Endpoints
+    with server.JsonEntitiesFromSchemas {
+
+  val routes: Route =
+    cors() {
+      artifact.implementedByAsync {
+        case ArtifactParams(maybeLanguageStr, maybePlatformStr) =>
+          val maybeLanguage = maybeLanguageStr.flatMap(Language.fromLabel)
+          val maybePlatform = maybePlatformStr.flatMap(Platform.fromLabel)
+          for (artifacts <- database.getAllArtifacts(maybeLanguage, maybePlatform))
+            yield {
+              val distinctArtifacts =
+                artifacts.map(artifact => ArtifactResponse(artifact.groupId.value, artifact.artifactId)).distinct
+              // TODO: The values below are placeholders, will need to populate them w. real data.
+              // See: https://github.com/scalacenter/scaladex/pull/992#discussion_r841500215
+              Page(
+                pagination = Pagination(
+                  current = 0,
+                  pageCount = 0,
```

Comment:
```suggestion
                  pageCount = 1,
```

I don’t know if pages start at 0 or 1, maybe `current` needs to be changed to `1` to?

Reply:
Looks like they start at 1, I'll update this 

https://github.com/scalacenter/scaladex/blob/d59165e8b31a19a9351df4bccdce83bb8ebb7245/modules/server/src/main/scala/scaladex/server/route/package.scala#L36